seed/seed.json

```diff
@@ -218,6 +218,34 @@
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
         },
+        {
+            "name": "SpeedreaderReleaseStudy",
+            "experiments": [
+                {
+                    "name": "Enabled",
+                    "probability_weight": 5,
+                    "feature_association": {
+                        "enable_feature": ["Speedreader"]
+                    }
+                },
+                {
+                    "name": "Disabled",
+                    "probability_weight": 95,
+                    "feature_association": {
+                        "disable_feature": ["Speedreader"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "92.0.4515.115",
```

Comment:
pls check other entries in this file, this has format chromium + brave, e.g. `92.1.28.xxxx`

Reply:
@moritzhaller maybe we could add a check for this field? e.g. second digit should always be 1, as in Brave

Reply:
fixed